<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersWindow</class>
 <widget class="QDialog" name="UsersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Пользователи</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IMG.qrc">
    <normaloff>:/IMG/IMG/logoSN.png</normaloff>:/IMG/IMG/logoSN.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 249, 255);</string>
  </property>
  <widget class="QListWidget" name="usersList">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>200</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget
{
	background-color: white;
}

QScrollBar:vertical 
{
	border: none;
    background: #f5f5f5;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical 
{
	background: #c1c1c1;
    min-height: 20px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover 
{
	background: #a8a8a8;
}

QScrollBar::handle:vertical:pressed
{
	background: #787878;
}

QScrollBar::add-line:vertical 
{
	border: none;
    background: none;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical 
{
	border: none;
	background: none;
	height: 0px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
	background: none;
}

QScrollBar:horizontal 
{
	border: none;
    background: #f5f5f5;
    height: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:horizontal 
{
    background: #c1c1c1;
    min-width: 20px;
	border-radius: 5px;
}

QScrollBar::handle:horizontal:hover 
{
	background: #a8a8a8;
}

QScrollBar::handle:horizontal:pressed 
{
	background: #787878;
}

QScrollBar::add-line:horizontal 
{
    border: none;
    background: none;
    width: 0px;
    subcontrol-position: right;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal 
{
    border: none;
    background: none;
    width: 0px;
    subcontrol-position: left;
	subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
	background: none;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchText">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>70</y>
     <width>431</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: white;
	color: black;
	border: 1px solid #4a148c; 
	border-radius: 20px;
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="placeholderText">
    <string>Поиск по логину</string>
   </property>
  </widget>
  <widget class="QLabel" name="countFriendsLabel">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>10</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="allUsersButton">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>130</y>
     <width>251</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	background-color: rgb(248, 249, 255);
    color: black;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
}

QPushButton:hover
 {
    background-color: rgb(229, 216, 255);
}

QPushButton:pressed 
{
    background-color: rgb(218, 188, 255);
}</string>
   </property>
   <property name="text">
    <string>Все пользователи</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>70</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
    color: white;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
    border: 1px solid #4a148c; 
}

QPushButton:hover
 {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7b1fa2,
        stop:1 #aa00ff
    );
    border: 2px solid #7b1fa2;
}

QPushButton:pressed 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #4a148c,
        stop:1 #7b1fa2
    );
}</string>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="friendsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	background-color: rgb(248, 249, 255);
    color: black;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
}

QPushButton:hover
 {
    background-color: rgb(229, 216, 255);
}

QPushButton:pressed 
{
    background-color: rgb(218, 188, 255);
}</string>
   </property>
   <property name="text">
    <string>Мои друзья</string>
   </property>
  </widget>
  <widget class="QLabel" name="friendsMessage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="text">
    <string>У вас нет друзей</string>
   </property>
  </widget>
  <widget class="QLabel" name="searchMessage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Пользователей не найдено</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="IMG.qrc"/>
 </resources>
 <connections/>
</ui>
